earth_enterprise/src/fusion/autoingest/geAssetRoot.cpp

```diff
@@ -19,7 +19,7 @@
 
 namespace geAssetRoot {
 
-std::string VolumeName = "assetroot";
+const char VolumeName[] = "assetroot";
```

Comment:
Minor: it's cleaner to put the memory fixes in a different PR (that applies to this and the other memory fixes included in this PR). I'm not sure if it's worth the effort to pull them out at this point, but it's something to keep in mind for the future. It's best to keep each PR focused on a single purpose.